src/ScriptEditor/ui/OptionsModal.tsx

```diff
@@ -13,14 +13,17 @@ import SaveIcon from "@mui/icons-material/Save";
 
 import { ThemeEditorModal } from "./ThemeEditorModal";
 
-interface IProps {
+type OptionsModalProps = {
   options: Options;
   save: (options: Options) => void;
   onClose: () => void;
   open: boolean;
-}
+};
+
+const CURSOR_STYLES: CursorStyle[] = ["line", "block", "underline", "line-thin", "block-outline", "underline-thin"];
+const CURSOR_BLINKING: CursorBlinking[] = ["blink", "smooth", "phase", "expand", "solid"];
```

Comment:
Just a note, I'm not usually a fan of these ALL_CAPS_CONSTANTS but it's fine here and I see why you couldn't really use an enum here which normally would have been more appropriate (type is determined externally so enum is harder to use).

Reply:
Caps here totally make sense to me - when I see non-caps variable when reading a React component code I assume it's either a prop or a comp-scoped constant (sometimes it's also some 3rd party, but usually those are functions).

Caps help me (and others, I hope) understand, that the value is not expected to be modified in any way. And I don't have to use IDE or scroll back and forth the source file to understand this. But of course, I don't use caps everywhere where I have a const. This is just a place where I feel it makes the most sense.

Reply:
Just a note about enums - quite often it's a better choice in a similar situation. An exception could be that you want to display options in a different order or to define just a subset of all possible values.
But yeah, in this specific case it does not really matter - if there was an enum provided by monaco, I'd use it.

Reply:
Thank you for the review by the way!
Like your PR comments :)